examples/deployment/scripts/transfer/asc-logic.js

```diff
@@ -0,0 +1,23 @@
+const {
+  transferMicroAlgosContract
+} = require('../src/asa-helpers');
+
+async function run(runtimeEnv, accounts, deployer) {
+
+  const goldOwnerAccount = deployer.accountsByName.get("gold-owner-account");
+  const johnAccount = deployer.accountsByName.get("john-account");
+
+  if (deployer.isDeployMode) {
+
+    const logicSignature = deployer.asc.get("3-gold-asc.teal").logicSignature;
+
+    // WIll pass
+    await transferMicroAlgosContract(deployer, goldOwnerAccount, johnAccount, 99, logicSignature);
+
+    // Gets rejected by logic
+    await transferMicroAlgosContract(deployer, goldOwnerAccount, johnAccount, 101, logicSignature);
```

Comment:
Why would you run it twice in the example? It should be better to run the whole example twice because the first time it shouldn't fail. Why would we have a failing example? Then it would be better to have a non-failing example too.

Reply:
![task](https://user-images.githubusercontent.com/26249903/91954893-0eb99080-ed20-11ea-91d6-46c581a5c22f.png)
It was in task

Reply:
ok. then it's fine.